3-networks-dual-svpc/modules/restricted_shared_vpc/private_service_connect.tf

```diff
@@ -16,12 +16,22 @@
 
 
 module "private_service_connect" {
-  source  = "terraform-google-modules/network/google//modules/private-service-connect"
-  version = "~> 9.0"
+  # TODO: Update reference to module version when service_directory_registrations
+  # is officially released. Only 1 commit has been made to the private service connect
+  # submodule since v9.0.0. This commit adds var.service_directory_region which is required
+  # to use the default region from  data.terraform_remote_state.bootstrap.outputs.common_config.default_region
+  # https://github.com/terraform-google-modules/terraform-google-network/commits/d4855d6b82b7963e22ed0f05aa31900915cfb954/modules/private-service-connect
+
+  # Example update:
+  # source  = "terraform-google-modules/network/google//modules/private-service-connect?ref=d4855d6b82b7963e22ed0f05aa31900915cfb954"
+  # version = "~> 9.1"
+  source = "github.com/terraform-google-modules/terraform-google-network//modules/private-service-connect?ref=d4855d6b82b7963e22ed0f05aa31900915cfb954"
```

Comment:
terraform-google-network v9.1.0 is available - can you please point to this release now?
https://github.com/terraform-google-modules/terraform-google-network/releases/tag/v9.1.0

Reply:
Thanks! Fixed in b8e66b6f27372508428ab90ebc0ce81f7e38e2df